.github/workflows/ci.yml

```diff
@@ -58,21 +73,15 @@ jobs:
       - name: Check that workflows are up to date
         run: sbt ++${{ matrix.scala }} githubWorkflowCheck
 
-      - name: Check formatting
-        run: sbt ++${{ matrix.scala }} scalafmtCheckAll scalafmtSbtCheck
-
-      - name: Check headers
-        run: sbt ++${{ matrix.scala }} headerCheckAll
-
-      - name: Compile
-        run: 'sbt ++${{ matrix.scala }} test:compile'
-
-      - name: Run tests
-        run: sbt ++${{ matrix.scala }} test
+      - run: sbt ++${{ matrix.scala }} ci
```

Comment:
Now that I see it from a workflow perspective, I'm not sure I consider this progress.  It is now harder to see on what step a build fails.

Reply:
Opened https://github.com/http4s/sbt-http4s-org/pull/120#discussion_r780729127 to think about this one.